### PR TITLE
Initialize blockhashes

### PR DESCRIPTION
**Pull request by LindaGuiga**
@BGluth This PR initializes the block hashes of the genesis block.

### CODE REVIEW

File: evm/src/fixed_recursive_verifier.rs [hunk 1]
```diff
@@ -1035,7 +1035,8 @@ where
         } else {
             // Initialize genesis_state_trie, state_root_after and the block number for correct connection between blocks.
```
Comment (Nashtare) [inline evm/src/fixed_recursive_verifier.rs:1036]: I think we're missing an update to the comment here

**Comment by BGluth** [comment]
I might not be the best to review plonky2 logic, but I'm excited for when this PR lands! :laughing:

**Comment by Nashtare** [comment]
@pgebheim added to the RC1 milestone as this is a bugfix for the block proof logic

**Review by hratoanina — APPROVED** [review]

**Comment by pgebheim** [comment]
Perfect thanks.

**Pull request merged**